src/webpack.config.js

```diff
@@ -114,6 +126,7 @@ module.exports = (env) => ({
   optimization: {
     minimize: isProduction,
     minimizer: [terserPlugin],
-    noEmitOnErrors: true,
+    emitOnErrors: false,
+    usedExports: false, // prevents orphaning entire bundle in production mode
```

Comment:
Could you elaborate a bit on this change? like how could it prevent orphaning bundle in production mode?

Reply:
The main entry of this configuration just exports stuff, but doesn't consume anything. With Webpack 5 and terser, it decides just to orphan everything because nothing is used. I tried to get terser to ignore the entry, but it didn't seem to work.

usedExports basically prevents tree shaking of exports that aren't actually consumed, which is not ideal but 1) the file isn't used by OSD (OSD uses lib directory) and 2) the outputted file is still smaller than it was before webpack 5, so I think I think this is okay - can figure out a better solution in the future (honestly, don't think anyone should be consuming a design system this way)